lib/dolly/document.rb

```diff
@@ -53,8 +53,8 @@ def rev= value
     def save
       self.doc['_id'] = self.id if self.id.present?
       self.doc['_id'] = self.class.next_id if self.doc['_id'].blank?
-      set_created_at if respond_to? :set_created_at
-      set_updated_at if respond_to? :set_updated_at
+      set_created_at if respond_to? :created_at
```

Comment:
these were not working correctly so I fixed them in here as well as saving a class without the `timestamps!` macro was throwing a no method error.